### PR TITLE
update Bloomberg Data and Analytics documentation and screenshots for…

### PR DESCRIPTION
**Pull request by Shrewbitz**
… August 2024 release

### CODE REVIEW

**Comment by Jak-MS** [comment]
@DougKlopfenstein 
Can you review the proposed changes? 

**Important**: When the changes are ready for publication, adding a `#sign-off` comment is the best way to signal that the PR is ready for the review team to merge. 

#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

**Comment by Shrewbitz** [comment]
Hey @DougKlopfenstein this documentation contains updated information about our updated PBI connector that was in the **August PBI release**. Is there anything else I need to do on my side?

**Review by DougKlopfenstein — APPROVED** [review]

**Comment by DougKlopfenstein** [comment]
#sign-off

**Comment by DougKlopfenstein** [comment]
Hi @Shrewbitz - it looks like you need to do the sign off. To do this sign off, just enter `#sign-off` in a comment. Once signed off, a final review will be done to the article and, if nothing else needs to be done to the article, it will be merged.

**Comment by Shrewbitz** [comment]
#sign-off

**Comment by DougKlopfenstein** [comment]
@Jak-MS - We've signed off, but the system has rejected both of our sign offs. Please review and merge the content of this PR.

File: powerquery-docs/connectors/bloomberg-data-and-analytics.md [hunk 1]
```diff
@@ -1,9 +1,9 @@
 ---
 title: Power Query Bloomberg Data and Analytics connector
 description: Provides basic information and prerequisites for the Bloomberg Data and Analytics connector, descriptions of the optional input parameters, and discusses limitations and issues you might encounter.
-author: kanstantsin-chernik
+author: Shrewbitz
```
Comment (Court72) [inline powerquery-docs/connectors/bloomberg-data-and-analytics.md:4]:
```suggestion
author: DougKlopfenstein
```

File: powerquery-docs/connectors/bloomberg-data-and-analytics.md [hunk 2]
```diff
@@ -34,36 +34,49 @@ To connect to Bloomberg Data and Analytics:
 
 1. In Power BI Desktop, select **Home** > **Get Data**.
 
-2. Select **Other** from the categories on the left, select **Bloomberg Data and Analytics**, and then select **Connect**.
+2. Select **Bloomberg Data and Analytics**, and then select **Connect**.
 
-   :::image type="content" source="./media/bloomberg-data-and-analytics/get-bql-data.png" alt-text="Get Data from Bloomberg Data and Analytics.":::
+   :::image type="content" source="./media/bloomberg-data-and-analytics/get-data.png" alt-text="Get data from Bloomberg Data and Analytics.":::
```
Comment (Court72) [inline powerquery-docs/connectors/bloomberg-data-and-analytics.md:39]:
```suggestion
   :::image type="content" source="./media/bloomberg-data-and-analytics/get-data.png" alt-text="Screenshot that shows get data from Bloomberg Data and Analytics.":::
```

File: powerquery-docs/connectors/bloomberg-data-and-analytics.md [hunk 3]
```diff
@@ -34,36 +34,49 @@ To connect to Bloomberg Data and Analytics:
...
 
-   :::image type="content" source="./media/bloomberg-data-and-analytics/bql-query.png" alt-text="Enter BQL query.":::
+   With the introduction of SQL support, you gain the benefit of [Query Folding](../query-folding-examples.md) in the Query Editor. This performance optimization, which reduces data transfer, is available exclusively for SQL queries.
 
-5. To sign in to your Bloomberg account, select **Sign in**.
+   > [!NOTE]
+   >The latest version of the connector introduces a unified query function, Bloomberg.Query(), in the formula bar. This function accepts both BQL and SQL, providing a streamlined experience. For existing reports developed with earlier versions, the BQL.Query() function remains operational and fully supported. Each function has its own authentication. To avoid logging in more than once, it is best practice to use Bloomberg.Query().
+   :::image type="content" source="./media/bloomberg-data-and-analytics/function-bar.png" alt-text="Function bar queries.":::
+
+   **4a. Bloomberg Query Language (BQL) Example:**\
```
Comment (Court72) [inline powerquery-docs/connectors/bloomberg-data-and-analytics.md:51]:
```suggestion
   - **Bloomberg Query Language (BQL) Example:**
   
```

File: powerquery-docs/connectors/bloomberg-data-and-analytics.md [hunk 4]
```diff
@@ -34,36 +34,49 @@ To connect to Bloomberg Data and Analytics:
...
+    To learn more about BQL, contact your Bloomberg Sales representative.\
```
Comment (Court72) [inline powerquery-docs/connectors/bloomberg-data-and-analytics.md:52]:
```suggestion
    To learn more about BQL, contact your Bloomberg Sales representative.
    
```

File: powerquery-docs/connectors/bloomberg-data-and-analytics.md [hunk 5]
```diff
@@ -34,36 +34,49 @@ To connect to Bloomberg Data and Analytics:
...
+    :::image type="content" source="./media/bloomberg-data-and-analytics/bql-example.png" alt-text="Example BQL query.":::
```
Comment (Court72) [inline powerquery-docs/connectors/bloomberg-data-and-analytics.md:53]:
```suggestion
    :::image type="content" source="./media/bloomberg-data-and-analytics/bql-example.png" alt-text="Screenshot of an example BQL query.":::
```

File: powerquery-docs/connectors/bloomberg-data-and-analytics.md [hunk 6]
```diff
@@ -34,36 +34,49 @@ To connect to Bloomberg Data and Analytics:
...
 
-   :::image type="content" source="./media/bloomberg-data-and-analytics/sign-in.png" alt-text="Select sign in button.":::
+    **4b. SQL Query Example:**\
```
Comment (Court72) [inline powerquery-docs/connectors/bloomberg-data-and-analytics.md:55]:
```suggestion
    - **SQL Query Example:**
    
```

File: powerquery-docs/connectors/bloomberg-data-and-analytics.md [hunk 7]
```diff
@@ -34,36 +34,49 @@ To connect to Bloomberg Data and Analytics:
...
+    :::image type="content" source="./media/bloomberg-data-and-analytics/sql-example.png" alt-text="Example SQL query.":::
+
+5. To sign in to your Bloomberg account, select **Sign in**.
 
-6. In the window that appears, provide your credentials to sign in to your Bloomberg account. If you entered an email address and a password, select **Next**.
+   :::image type="content" source="./media/bloomberg-data-and-analytics/sign-in.png" alt-text="Select sign in button":::
```
Comment (Court72) [inline powerquery-docs/connectors/bloomberg-data-and-analytics.md:60]:
```suggestion
   :::image type="content" source="./media/bloomberg-data-and-analytics/sign-in.png" alt-text="Screenshot that shows the sign in button.":::
```

File: powerquery-docs/connectors/bloomberg-data-and-analytics.md [hunk 8]
```diff
@@ -34,36 +34,49 @@ To connect to Bloomberg Data and Analytics:
...
+6. In the window that appears, provide your credentials to sign in to your Bloomberg account. Enter a terminal username and password, then select **Next**.
 
-   :::image type="content" source="./media/bloomberg-data-and-analytics/bloomberg-sign-in.png" alt-text="Sign in to Bloomberg.":::
+    :::image type="content" source="./media/bloomberg-data-and-analytics/bb-sign-in.png" alt-text="Sign in to Bloomberg.":::
```
Comment (Court72) [inline powerquery-docs/connectors/bloomberg-data-and-analytics.md:63]:
```suggestion
    :::image type="content" source="./media/bloomberg-data-and-analytics/bb-sign-in.png" alt-text="Screenshot that shows how to sign in to Bloomberg.":::
```

File: powerquery-docs/connectors/bloomberg-data-and-analytics.md [hunk 9]
```diff
@@ -34,36 +34,49 @@ To connect to Bloomberg Data and Analytics:
...
 
-7. Enter your B-Unit code and select **Log In**.
+7. Enter your Bloomberg B-Unit Mobile App code or B-Unit code and select **Submit**.
 
-   :::image type="content" source="./media/bloomberg-data-and-analytics/bunit.png" alt-text="Enter your BUnit code.":::
+    :::image type="content" source="./media/bloomberg-data-and-analytics/mobile-app.png" alt-text="Enter your auth code.":::
```
Comment (Court72) [inline powerquery-docs/connectors/bloomberg-data-and-analytics.md:67]:
```suggestion
    :::image type="content" source="./media/bloomberg-data-and-analytics/mobile-app.png" alt-text="Screenshot that shows where to enter your auth code.":::
```

File: powerquery-docs/connectors/bloomberg-data-and-analytics.md [hunk 10]
```diff
@@ -34,36 +34,49 @@ To connect to Bloomberg Data and Analytics:
...
 
 8. Once you've successfully signed in, select **Connect**.
 
-   :::image type="content" source="./media/bloomberg-data-and-analytics/connect.png" alt-text="Signed in and ready to connect.":::
+   :::image type="content" source="./media/bloomberg-data-and-analytics/sign-in.png" alt-text="Select connect button.":::
```
Comment (Court72) [inline powerquery-docs/connectors/bloomberg-data-and-analytics.md:71]:
```suggestion
   :::image type="content" source="./media/bloomberg-data-and-analytics/sign-in.png" alt-text="Screenshot that shows the connect button.":::
```

File: powerquery-docs/connectors/bloomberg-data-and-analytics.md [hunk 11]
```diff
@@ -34,36 +34,49 @@ To connect to Bloomberg Data and Analytics:
...
 
-Once the connection is established, you will see data available for preview in **Navigator**.
+9. After the connection is established, you will see data available for preview in the **Navigator**.
 
-:::image type="content" source="./media/bloomberg-data-and-analytics/navigator-view.png" alt-text="Select tables in Navigator View.":::
+   **9a. Bloomberg Query Language (BQL) Example:**\
```
Comment (Court72) [inline powerquery-docs/connectors/bloomberg-data-and-analytics.md:75]:
```suggestion
   - **Bloomberg Query Language (BQL) Example:**
   
```

File: powerquery-docs/connectors/bloomberg-data-and-analytics.md [hunk 12]
```diff
@@ -34,36 +34,49 @@ To connect to Bloomberg Data and Analytics:
...
+   :::image type="content" source="./media/bloomberg-data-and-analytics/bql-load.png" alt-text="Select BQL tables in the navigator view.":::\
```
Comment (Court72) [inline powerquery-docs/connectors/bloomberg-data-and-analytics.md:76]:
```suggestion
   :::image type="content" source="./media/bloomberg-data-and-analytics/bql-load.png" alt-text="Screenshot that shows where to select BQL tables in the navigator view.":::

```

File: powerquery-docs/connectors/bloomberg-data-and-analytics.md [hunk 13]
```diff
@@ -34,36 +34,49 @@ To connect to Bloomberg Data and Analytics:
...
+   **9b. SQL Query Example:**\
```
Comment (Court72) [inline powerquery-docs/connectors/bloomberg-data-and-analytics.md:77]:
```suggestion
   - **SQL Query Example:**
   
```

File: powerquery-docs/connectors/bloomberg-data-and-analytics.md [hunk 14]
```diff
@@ -34,36 +34,49 @@ To connect to Bloomberg Data and Analytics:
...
+   :::image type="content" source="./media/bloomberg-data-and-analytics/sql-load.png" alt-text="Select SQL tables in the navigator view.":::
```
Comment (Court72) [inline powerquery-docs/connectors/bloomberg-data-and-analytics.md:78]:
```suggestion
   :::image type="content" source="./media/bloomberg-data-and-analytics/sql-load.png" alt-text="Screenshot that shows where to select SQL tables in the navigator view.":::
```

File: powerquery-docs/connectors/bloomberg-data-and-analytics.md (hunk 1 above)
Comment (Shrewbitz) [reply powerquery-docs/connectors/bloomberg-data-and-analytics.md:4]: Does author mean something different than the person who wrote the documentation in this instance?

Comment (Court72) [reply powerquery-docs/connectors/bloomberg-data-and-analytics.md:4]: It usually matches the ms.author, but it doesn't have to. We can keep it as-is.

**Pull request merged**